tdp/cli/params/plan/force_option.py

```diff
@@ -12,6 +12,8 @@ def force_option(func: FC) -> FC:
     """
     return click.option(
         "--force",
+        envvar="FORCE_PLAN",
```

Comment:
All other tdp env vars are prefixed by `TDP_`. The naming is also a bit confusing.

```suggestion
        envvar="TDP_FORCE_PLAN_OVERRIDE",
```